.github/workflows/test-conda.yml

```diff
@@ -0,0 +1,56 @@
+name: Test using Conda environments
+
+on: [push, pull_request]
+
+jobs:
+  TestConda:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        channel: ['']
+        dev: [false]
+        geos: ['geos']
+        numpy: ['numpy']
+        python: [3.6, 3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Conda
+        uses: s-weigand/setup-conda@v1.0.5
+        with:
+          activate-conda: false
+          conda-channels: ${{ matrix.channel }}
+          python-version: ${{ matrix.python }}
+
+      - name: Setup Environment
+        shell: bash
+        run: |
+          conda create --name test python=${{ matrix.python}} conda ${{ matrix.geos }} ${{ matrix.numpy }}
```

Comment:
the `conda` as python package to install shouldn't be there I think?

Reply:
Right, had added that back in to match the Travis config while trying to get things working, but it was a different issue.  Taking back out now.